fighthealthinsurance/ml/ml_citations_helper.py

```
@@ -194,14 +194,18 @@ async def generate_generic_citations(
+                try:
+                    result = (
+                        await best_within_timelimit(
+                            partial_awaitables,
+                            score_fn=cls.make_score_fn(),
+                            timeout=timeout,
+                        )
+                        or []
                     )
-                    or []
-                )
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)